develop/smart-contracts/precompiles/interact-with-precompiles.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Interact with Precompiles
+title: Interact With Precompiles
```

Comment:
Actually it should be lowercase. We use chicago style. I always use this title case checker when in doubt (just make sure its set to chicago style): https://titlecaseconverter.com/